src/ert/shared/share/ert/forward-models/old_style/MAKE_DIRECTORY

```diff
@@ -1,5 +1,5 @@
-STDERR    make_directory.stderr
-STDOUT    make_directory.stdout
+STDERR    logs/make_directory.stderr
```

Comment:
This is the forward model, so it will be in the run path, any reason to put it in a logs folder?

Reply:
First of all, do you know why these files are created and what they are used for even when creating of directories works as intended?

Reply:
Well, no idea, they are probably empty always? Is there an option of saying `STDERR NULL`? of dev/null?

Reply:
I just tried removing both lines and the forward model still works fine but does not create these files.
Should we just get rid of them altogether you think?

Reply:
Sounds good, I'll close this PR.
Thanks